static/js/containers/ProfilePage_test.js

```diff
@@ -73,23 +73,25 @@ describe("ProfilePage", function() {
     helper.cleanup();
   });
 
-  let confirmSaveButtonBehavior = (updatedProfile, pageElements, validationFailure=false) => {
+  let confirmSaveButtonBehavior = (updatedProfile, pageElements, validationFailure=false, actions = []) => {
     let { div, button } = pageElements;
     button = button || div.querySelector(nextButtonSelector);
     patchUserProfileStub.throws("Invalid arguments");
     patchUserProfileStub.withArgs(SETTINGS.username, updatedProfile).returns(Promise.resolve(updatedProfile));
 
-    let actions = [];
-    if (!validationFailure) {
+    if ( actions === [] ) {
```

Comment:
This won't compare since the `[]` and `actions` will always be different objects. Can you check the `length` instead?